### PR TITLE
ERADICATES Soul Contracts from the code

### PR DESCRIPTION
**Pull request by solwashere**
# Document the changes in your pull request

Devil is no longer maintained, and soul contracts do not fit in with the current direction that the lore team is moving in.

# Spriting

# Wiki Documentation

# Changelog

:cl:  
  
rscdel: soul contracts go bye bye

/:cl:

### CODE REVIEW

**Comment by solwashere** [comment]
okay I might have to fix the maps

**Comment by solwashere** [comment]
Toying with the idea of just rewording the contract

**Review by patpol4 — CHANGES_REQUESTED** [review]
Please dont remove them from the map but instead replace with the plain version of the cabinet with no code attached

**Comment by neeshacark** [comment]
i mean this is just going to buff devil which like... ok

**Comment by solwashere** [comment]
> i mean this is just going to buff devil which like... ok

The unmaintained and broken game mode that isn't even in rotation?

**Review by patpol4 — APPROVED** [review]
Map fine, awaiting balance/code approval

**Comment by MajManatee** [comment]
cringe

**Comment by solwashere** [comment]
> cringe

Do you even care about lore or a broken game mode?

File: code/modules/paperwork/photocopier.dm [hunk 1]
```diff
@@ -323,11 +323,6 @@
 	
 /obj/machinery/photocopier/proc/copy(var/obj/item/paper/copy)
 	var/copy_as_paper = TRUE
-	if(istype(copy, /obj/item/paper/contract/employment))
-		var/obj/item/paper/contract/employment/E = copy
-		var/obj/item/paper/contract/employment/C = new /obj/item/paper/contract/employment (loc, E.target.current)
-		if(C)
-			copy_as_paper = FALSE
 	if(copy_as_paper)
```
Comment (Moltijoe) [inline code/modules/paperwork/photocopier.dm]: can remove copy_as_paper as it's only used to exclude employment contracts

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/photocopier.dm (hunk 1 above)
Comment (solwashere) [reply code/modules/paperwork/photocopier.dm]: Will get to it when I'm off work

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**